jcl/src/java.base/share/classes/openj9/internal/criu/InternalCRIUSupport.java

```diff
@@ -121,19 +151,37 @@ public static boolean enableCRIUSecProvider() {
 
 	/**
 	 * Queries if CRIU Checkpoint is allowed.
-	 * isCRIUSupportEnabled() is invoked first to check if CRIU support is enabled,
+	 * isCRaCorCRIUSupportEnabled() is invoked first to check if CRac or CRIU support is enabled,
```

Comment:
Typo: CRac -> CRaC.

Reply:
Fixed in https://github.com/eclipse-openj9/openj9/pull/18815